src/Development/IDE/Spans/Type.hs

```diff
@@ -29,15 +29,17 @@ data SpanInfo =
            ,spaninfoEndCol :: {-# UNPACK #-} !Int
             -- ^ End column of the span (absolute), zero-based.
            ,spaninfoType :: !(Maybe Type)
-            -- ^ A pretty-printed representation fo the type.
+            -- ^ A pretty-printed representation for the type.
            ,spaninfoSource :: !SpanSource
            -- ^ The actutal 'Name' associated with the span, if
             -- any. This can be useful for accessing a variety of
             -- information about the identifier such as module,
             -- locality, definition location, etc.
            }
 instance Show SpanInfo where
-  show (SpanInfo sl sc el ec t n) = show [show sl, show sc, show el, show ec, show $ isJust t, show n]
+  show (SpanInfo sl sc el ec t n) =
+    intercalate " " ["(SpanInfo", show sl, show sc, show el, show ec
```

Comment:
This can be `unwords` rather than `intercalate " "`.

Reply:
Of course. Funny how the route you take influences what you end up with.

Fixed.